.env-example

```diff
@@ -0,0 +1,3 @@
+GITHUB_PRIVATE_KEY=""
```

Comment:
Are the quotes necessary here? 🤔

Reply:
@swinton the quotes are necessary. The quickstart guide explains why. Quotes are needed for multi-line variables, for the PEM key.